Prog/Cvd19_affctd_ind_pop_tables.sas

```diff
@@ -0,0 +1,678 @@
+  table 
+
+    /** Rows **/
+    all='Total' statefip=' ' * ( all=' ' upuma=' ' ),
+
+    /** Columns **/
+    sum='Households' * total=' ' * f=comma12.0
+
+    sum='Annual household income ($ 2018), 2014-18' * 
+    ( inctot_sum='Total income' 
+      incearn_sum='Earnings' 
+      cvd19_affctd_incearn_sum='Earnings from COVID-19 affected industries' )
+
+    pctsum<inctot_sum>='COVID-19 affected earnings as pct. total income' * 
+    cvd19_affctd_incearn_sum=' ' *
+    f=comma12.1
+
+    pctsum<incearn_sum>='COVID-19 affected earnings as pct. total earnings' * 
+    cvd19_affctd_incearn_sum=' ' *
+    f=comma12.1
+  ;
+  format upuma $upuma_to_mwcog_jurisd.;
+run;
+
+title3 'Households with workers in COVID-19 affected industries by potential change in HUD income category, MWCOG region';
+
+proc tabulate data=Covid19.cvd19_affctd_ind_pop format=comma16.0 noseps missing;
+  where mwcog_region = 1 and pernum = 1 and incearn_sum > 0 and not( missing( hud_inc ) );
+  weight hhwt;
+  class statefip hud_inc hud_inc_cvd19;
+  var total;
+  table 
+  
+    /** Pages **/
+    all='MWCOG region' statefip=' ',
+
+    /** Rows **/
+    all='Total' hud_inc=' ',
+
+    /** Columns **/
+    n='N (unwtd)' * total=' ' * f=comma8.0
+    sum='Households' * total=' ' * f=comma12.0
+
+    sum='By HUD income category without COVID-19 affected earnings' * total=' ' * hud_inc_cvd19=' '
+    
+    /box = 'By HUD income category, 2014-18' condense
+  ;
+run;
+
+
+title3 'Households with workers in COVID-19 affected industries by potential change in housing cost burden, MWCOG region';
+
+proc tabulate data=Covid19.cvd19_affctd_ind_pop format=comma16.0 noseps missing;
+  where mwcog_region = 1 and pernum = 1 and incearn_sum > 0 and not( missing( hud_inc ) );
+  weight hhwt;
+  class statefip hsg_cost_ratio hsg_cost_ratio_cvd19 / order=data preloadfmt;
+  var total;
+  table 
+
+    /** Pages **/
+    all='MWCOG region' statefip=' ',
+
+    /** Rows **/
+    all='Total' hsg_cost_ratio=' ',
+
+    /** Columns **/
+    n='N (unwtd)' * total=' ' * f=comma8.0
+    sum='Households' * total=' ' * f=comma12.0
+
+    sum='By housing cost burden without COVID-19 affected earnings' * total=' ' * hsg_cost_ratio_cvd19=' '
+    
+    /box = 'By housing cost burden, 2014-18' condense
+  ;
+  format hsg_cost_ratio hsg_cost_ratio_cvd19 hsg_cost_ratio.;
+run;
+
+
+**** Graphics ****;
+
+footnote2;
+footnote3 ' ';
+footnote4 'DRAFT - NOT FOR CITATION OR RELEASE';
+
+title3 'Workers in COVID-19 affected industries by annual earnings, MWCOG region';
+
+proc sgplot data=Covid19.cvd19_affctd_ind_pop;
+  where mwcog_region = 1 and cvd19_affctd_ind = 1 and &CVD19_BOT_1PCT_EARNINGS < incearn < &CVD19_TOP_1PCT_EARNINGS;
+  histogram incearn / weight=perwt;
+  label incearn = 'Annual earnings ($ 2018), 2014-18';
+run;
+
+
+title3 'Households with workers in COVID-19 affected industries by potential annual income loss, MWCOG region';
+
+/*** NEED TO TRIM OUTLIERS FROM THIS CHART ***
```

Comment:
Are you still planning to produce this chart? Or does the one right below remove the outliers? If it does you can delete the now unneeded code.